app/views/_create-secret.html

```diff
@@ -0,0 +1,267 @@
+<!-- Error messages -->
+<parse-error error="error" ng-if="error"></parse-error>
```

Comment:
no reason, I thought that its meant to the dialog, since we use it in YAML editor.. will replace with `alerts`